tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -58,7 +59,8 @@ def run(self):
             )
 
         all_ssh_jobs_done = False
-        while not all_ssh_jobs_done:
+        count = 0
```

Comment:
I feel if there is any issue while copying, ansible will definitely return the error and we are raising exception with this error message.
I don't think we will land in a situation where this utility reports success and in reality the password-less ssh is not working(If at all such a situation arises, then we definitely will add the checks that you have mentioned)